packages/formation/js/accordion.js

```diff
@@ -26,9 +26,13 @@ const loadAccordionHandler = () => {
 
   usaAccordionContentElements.forEach(el => {
     if (el.getAttribute('aria-hidden') === null) {
+      const buttonElement = el.parentElement.querySelector(
```

Comment:
could use `querySelectorAll` for some of this logic to make it more declarative. The query is kind of long but this logic is also really complicated.

```
// creates exactly the array you're looking for without using spread operator 
const usaAccordion = document.querySelectorAll('.usa-accordion, .usa-accordion-bordered')

// inverts the logic a little- looks for the button first and then queries the siblings
// the matching is done with the query e.g. where button's aria-expanded is true and .usa-accordion-content is not false 
document
.querySelectorAll('button[aria-controls][aria-expanded='true'] ~ .usa-accordion-content:not[aria-hidden='false')
.forEach(el => el.setAttribute(
	        'aria-hidden',
	        false,
	      );

document
.querySelectorAll('button[aria-controls][aria-expanded='false'] ~ .usa-accordion-content:not[aria-hidden='true')
.forEach(el => el.setAttribute(
	        'aria-hidden',
	        true,
	      );
```

I think this is mostly a matter of taste- I think query selectors for this kind of thing are easier to understand than a few lines of imperative code. Maybe @jbalboni has an opinion. 

Not totally relevant but `querySelectorAll` is supposed to be highly optimized.

Reply:
I didn't know you can even do that. Thanks